creme/metrics/_confusion_matrix.pyx

```diff
@@ -0,0 +1,232 @@
+import warnings
+
+import numpy as np
+cimport numpy as np
+
+from collections import defaultdict
+
+cdef bint boolean_variable = True
+
+cdef class ConfusionMatrix:
```

Comment:
How about if we move the methods from `ClassificationPerformanceEvaluator` into this class?

Reply:
Definetly, I will explore this option.